.github/workflows/test.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         id: cpu-cores
         uses: SimenB/github-actions-cpu-cores@v1
       - name: run tests
+        timeout-minutes: 20
```

Comment:
do we need these?

Reply:
Probably not now I fixed the other issues